pygame/time.py

```diff
@@ -112,7 +113,7 @@ def get_ticks():
 @ffi.callback("SDL_NewTimerCallback")
 def _timer_callback(interval, param):
     if sdl.SDL_WasInit(sdl.SDL_INIT_VIDEO):
-        event = ffi.new("SDL_Event")
+        event = ffi.new("SDL_Event*")
```

Comment:
Do we need to hang on to `event` somehow to avoid the memory allocated here being freed?